pkgs/tools/misc/phpDocumentor/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, makeWrapper, lib, php }:
+let
+  pname = "phpDocumentor";
```

Comment:
There's no need to have those in a `let`...`in` statement. You can directly add them in `stdenv.mkDerivation`.
If you need to use `${version`} them multiple times, add the `rec` keyword just after `stdenv.mkDerivation`.

Take inspiration from: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/php-packages/grumphp/default.nix

Reply:
Now it's perfectly fine, the latest changes are ok.